### DISCUSSION TITLE
Loading a prompt from an image doesn't work

### DISCUSSION THREAD

**Discussion by mald55** [MoonRide303/Fooocus-MRE#67] [Q&A] [answered]
I have noticed that when I load an image with metada in it to autofill the prompts it doesn't work. I also noticed that custom settings are not saved between sessions. Are these features missing or is this a bug of sorts?

**Comment by barepixels** [comment]
load an image with metada in it to autofill the prompts works here

new sessions load my preferred settings from "settings.json" file

**Reply by mald55** [reply]
I don't think a follow, I have to manually enable this setting in a file called 'settings.json'? 

Where is this file located?

**Comment by barepixels** [comment]
1) Duplicate "settings-example.json"
2) Change the new file name to "setting.json"
3) Edit "setting.json"

**Reply by mald55** [reply]
What's the path within the Fooocus-MRE-v1.0.45 folder I need to drop this file in?

**Comment by barepixels** [comment]
Find "settings-example.json"  put "setting.json" next to it.  
You can do the same for "paths-example.json" if you want to place your models, loras, or output folder somewhere else

**Reply by mald55** [reply, marked as answer]
I figured it out, thank you. 

For all those interested:
1-) go to X:\X\Fooocus-MRE-v1.0.45\Fooocus-MRE 
2-) Find a file called "settings-example.json", make a copy of it, and rename it to "setting.json"
3-) Open "setting.json" and change the settings to whatever you would like and save the file.

From now on everytime you open focus it will load with those settings you saved on the "setting.json" file.

If you don't know what settings to use simply look it up on the Wiki, or test them on the GUI and use the same numbers inside the "setting.json" file.
